docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/qualys-vmdr-source.md

```diff
@@ -104,3 +113,7 @@ Sometimes the asset information from the computer inventory data can exceed the
 
 - The `openPortListData` key only contains information about ports open since the last time computer asset was ingested instead of listing all open port history from all time.
 - The `softwareListData` is reduced down from the full details to simply a list/array of software names using the full name.
+
+### How can I differentiate between data types collected?
+
+The Qualys VMDR C2C collects all data using JSON as the format. It will add an additional key to your data called `LogType` with the values of `vulnerabilityLogs`, `knowledgeBaseLogs` and `assetInventory`. This allows you to easily filter between them in your search queries.
```

Comment:
```suggestion
The Qualys VMDR C2C collects all data using JSON as the format. It will add an additional key to your data called `LogType` with the values of `vulnerabilityLogs`, `knowledgeBaseLogs`, and `assetInventory`. This allows you to easily filter between them in your search queries.
```